pkg/discovery/dtofactory/node_entity_dto_builder.go

```diff
@@ -163,6 +162,10 @@ func (builder *nodeEntityDTOBuilder) BuildEntityDTOs(nodes []*api.Node) []*proto
 			continue
 		}
 
+		entityDto.ProviderPolicy = &proto.EntityDTO_ProviderPolicy{AvailableForPlacement: &isAvailableForPlacement}
+		if !isAvailableForPlacement {
+			glog.Warningf("Node %s has been marked unavailable for placement.", node.GetName())
```

Comment:
Maybe we can be more clear, saying something like `Node %s has been marked unavailable for placement due to disk pressure`?

Reply:
Updated!